src/webmon_app/reporting/templates/base.html

```diff
@@ -6,6 +6,23 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <title>{% block title %}Data Processing Report{% endblock %}</title>
 
+<!-- Digital Analytics Program (DAP) -->
+<script type="text/javascript">
+(function() {
+    var dapScript = document.createElement('script');
+    dapScript.id = '_fed_an_ua_tag';
+    dapScript.async = true;
+    dapScript.type = 'text/javascript';
+    dapScript.src = 'https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=DOE&subagency=BES&sitetopic=science&siteplatform=drupal';
```

Comment:
Is `siteplatform` mandatory? And do you know what the possible values are except `drupal`? If there is not a suitable value and it's not mandatory, I would leave it out.